crate/content.md

```diff
@@ -1,82 +1,138 @@
 # What is Crate?
 
-Crate allows to query and compute data with SQL in real time by providing a distributed aggregation engine, native search and super simple scalability. It offers auto-sharding and replication, super-fast multi index queries, distributed aggregations and sort, superfast full-text search, and super simple cluster management.
+Crate is a fast, scalable, easy to use SQL database that plays nicely
+with containers like Docker. It feels like the SQL databases you know,
+however makes scaling and operating your database ridiculously easy --
+regardless of the volume, complexity, or type of data. Crate is open
+source. It ingests millions of records per second for time series
+setups and delivers analytics results in subsecond real time.
+
+It comes with a distributed sort and aggregation engine, fast multi
+index queries, native full-text search and super simple scalability
+with sharding and partitioning builtin already. Preconfigured
+replication takes care of data resiliency. The cluster management can
+easily supervised with its builtin admin UI. Crate's masterless
+architecture and simplicity make the data part of Docker environments
+easy and elegant.
+
+Crate provides several installation packages, including a supported
+Docker image. It fits perfectly in an orchestrated microservices
+environment. It acts like an ether, an omnipresent, persistent layer
+for data. This way, application containers access their data
+regardless on which host the data nodes run.
 
 [Crate](https://crate.io/)
 
 %%LOGO%%
 
 ## How to use this image
 
+To form a cluster, just start the Crate container a few times in the
+background:
```

Comment:
this will start a couple of containers on your machine - which discover each other via multicast. in a production environment you'd run crate on different machines.